source/assets/sass/components/_playground.scss

```diff
@@ -0,0 +1,321 @@
+    'header' min-content
+    'main' 1fr
+    'footer' min-content / 100%;
+  min-height: 100vh;
+  overflow: hidden auto;
+
+  @include breakpoints.sl-breakpoint--large {
+    height: 100vh;
+    overflow: hidden;
+  }
+
+  #main-content {
+    display: inherit;
+    overflow: inherit;
+  }
+}
+
+.sl-r-banner--playground-grid {
+  align-items: center;
+  display: grid;
+  grid-template:
+    'logo title copy' auto
+    'alert alert alert' minmax(0, auto) / 1fr 1fr 1fr;
+
+  @include breakpoints.sl-breakpoint--large {
+    grid-template: 'logo title copy' auto / 1fr 1fr 1fr;
+  }
+}
+
+// add padding to items in order for alert to be full width
+.sl-r-banner--playground__brand {
+  padding-left: var(--sl-gutter);
+
+  @include breakpoints.sl-breakpoint--small {
+    padding-left: var(--sl-gutter--double);
+  }
+}
+
+.sl-l-section--playground {
+  --sl-inline-padding--container: 0;
+  --sl-inline-padding--container-large: 0;
+
+  border-bottom: 1px solid var(--sl-color--iron);
+  display: inherit;
+  overflow: inherit;
+  width: 100%;
+}
+
+.sl-r-banner__playground-title {
+  // Use smaller sizes for condensed header
+  --h1-size: var(--sl-font-size--xx-large);
+  --h1-size-lg: var(--sl-playground-heading);
+
+  margin-bottom: 0;
+}
+
+// Playground banner actions area
+.sl-r-banner__playground-actions {
+  gap: var(--sl-gutter);
+  display: contents;
+
+  @include breakpoints.sl-breakpoint--large {
+    display: grid;
+    grid-template: 'alert copy' auto / minmax(0, auto) max-content;
+  }
+}
+
+.sl-r-banner__playground-button {
+  grid-area: copy;
+  padding-right: var(--sl-gutter);
+  text-align: right;
+
+  @include breakpoints.sl-breakpoint--small {
+    padding-right: var(--sl-gutter--double);
+  }
+}
+
+// Playground banner actions alert message
+.sl-r-banner__playground-alert {
+  --sl-padding-block--alert: 0;
+
+  grid-area: alert;
+  height: 0;
+  opacity: 0;
+  padding-left: var(--sl-gutter--half);
+  padding-right: var(--sl-gutter--half);
+  transition: opacity 0.3s 0.05s, visibility 0.3s;
+  visibility: hidden;
+
+  &.show {
+    --sl-padding-block--alert: var(--sl-gutter--minus);
+
+    height: 100%;
+    opacity: 1;
+    visibility: visible;
+  }
+}
+
+// Playground Editor area
+.sl-c-playground {
+  --sl-block-margin--callout: 0;
+  --sl-block-padding--callout: var(--sl-gutter--quarter);
+
+  display: grid;
+  gap: var(--sl-gutter);
+  grid-template:
+    'sass' 1fr
+    'css' auto
+    'debug' auto / 100%;
+
+  @include breakpoints.sl-breakpoint--large {
+    // magic numbers, the layout is well balanced with these proportions
+    grid-template:
+      'sass css' minmax(20ex, 1fr)
+      'sass debug' minmax(0, 40%) / 1fr 1fr;
+    height: 100%;
+    overflow: inherit;
+  }
+}
+
+.sl-c-playground__panel {
+  display: grid;
+  overflow: inherit;
+  grid-template:
+    'editor-tabbar' min-content
+    'editor-error' min-content
+    'editor-code' 1fr / 1fr;
+}
+
+.sl-c-playground__panel-content {
+  grid-area: editor-code;
```

Comment:
When we have named areas like this, I'd have some preference for making sure each grid item is assigned to an area (like this one is, but others aren't) – even when it's not technically necessary for the layout. By assigning each element to an area, we can ensure that changes to the grid-template have the desired impact.

Reply:
They are all using a named grid area, but they are lower in the file... 

`editor-tabbar` is line 285 currently
`editor-error` is 241

Is that what you mean @mirisuzanne?

Reply:
oh, yes, ok - I missed that. 👍🏼